test/csg-conversions.js

```diff
@@ -0,0 +1,16 @@
+import test from 'ava';
+import {CSG} from '../csg';
+import {OBJ} from '../helpers/obj-store';
+import {assertSameGeometry} from '../helpers/asserts';
+
+test("CSG can be packed into and retrieved from a compact binary", t => {
```

Comment:
Maybe we should add documentation to explain that compactBinary cannot be JSONified, for reasons I don't know (UInt32Array issue ?)